vyxal/elements.py

```diff
@@ -1021,14 +1018,13 @@ def fibonaacis(_, ctx):
     An infinite lazylist of fibonaaci numbers
     """
 
-    @lazylist
     def gen():
         i = 0
         while True:
             yield sympy.fibonacci(i + 1)
             i += 1
 
-    return gen()
+    return LazyList(gen(), isinf=False)
```

Comment:
Why specifically say `isinf=False` when that's the default?

Reply:
Typo

Reply:
Should be true